packages/engine-multi/test/integration.test.ts

```diff
@@ -151,19 +151,20 @@ test.serial('trigger workflow-log for job logs', (t) => {
 
     let didLog = false;
 
-    api.execute(plan, emptyState).on('workflow-log', (evt) => {
-      if (evt.name === 'JOB') {
-        didLog = true;
-        t.deepEqual(evt.message, JSON.stringify(['hola']));
-        t.pass('workflow logged');
-      }
-    });
-
-    api.execute(plan, emptyState).on('workflow-complete', (evt) => {
-      t.true(didLog);
-      t.falsy(evt.state.errors);
-      done();
-    });
+    api
+      .execute(plan, emptyState)
+      .on('workflow-log', (evt) => {
+        if (evt.name === 'JOB') {
+          didLog = true;
+          t.deepEqual(evt.message, JSON.stringify(['hola']));
+          t.pass('workflow logged');
+        }
+      })
+      .on('workflow-complete', (evt) => {
+        t.true(didLog);
+        t.falsy(evt.state.errors);
+        done();
+      });
```

Comment:
@josephjclark 
Is this a good fix to the flaky test?

I think so. we basically want to check whether workflow-log happens before workflow-complete. Having two different execute meant we couldn't determine the order in which the events were fired because we didn't know which one executes faster. with a single execute, it's quite deterministic.

Reply:
Hey - oh , yes! Man this test was so dumb :joy: 

Yeah you're absolutely right. This is an editing error or just a straight up fail by me.

Thank you!